docs/docs/sourcing-from-hosted-services.md

```diff
@@ -14,4 +14,8 @@ There are many Gatsby plugins that integrate with SaaS (Software as a Service).
 
 These plugins will pull data from a hosted service and make it available to your Gatsby site via GraphQL.
 
+More examples are described on the Gatsby blog:
+
+- [Google Sheets](/blog/2019-07-23-google-sheets-gatsby-acroyoga-video-explorer/)
```

Comment:
FYI: The top-downloaded [google sheets plugin](https://www.gatsbyjs.org/packages/gatsby-source-google-sheets/?=sheets) doesn't look maintained, so I didn't link to it in the list..the blog post seemed to provide better guidance.